core/src/test/scripts/unit_tests_validate_data.py

```diff
@@ -699,7 +699,11 @@ def test_valid_discrete_cna(self):
             self.assertLessEqual(record.levelno, logging.INFO)
 
     def test_repeated_gene(self):
-        """Test if a warning is issued and the line is skipped if duplicate."""
+        """Test if a warning is issued and the line is skipped if duplicate. In the test data, the Entrez ID in line 6
```

Comment:
Python tooling expects docstrings to consist of a single-line descriptive sentence, optionally followed by a blank line and any paragraphs of additional details; see [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings). Could you add the blank line? That would, for instance, make the failure output of this test more sensible:

```Console
======================================================================
FAIL: test_repeated_gene (__main__.FeatureWiseValuesTestCase)
Test if a warning is issued and the line is skipped if duplicate. In the test data, the Entrez ID in line 6
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./unit_tests_validate_data.py", line 712, in test_repeated_gene
    self.assertEqual(len(record_list), 1)
AssertionError: 3 != 1

Stdout:
WARNING: data_cna_duplicate_gene.txt: line 6: Duplicate line for a previously listed feature/gene, this line will be ignored.; value encountered: '116983 (already defined on line 2)'
ERROR: data_cna_duplicate_gene.txt: line 6: column 5: Invalid CNA value: possible values are [-2, -1.5, -1, 0, 1, 2, NA]; value encountered: 'spam'
ERROR: data_cna_duplicate_gene.txt: line 6: column 7: Invalid CNA value: possible values are [-2, -1.5, -1, 0, 1, 2, NA]; value encountered: '3'
```

Reply:
Thanks, done